pyproject.toml

```diff
@@ -9,7 +9,7 @@ readme = "README.md"
 python = "^3.7"
 py7zr = "^0.17.4"
 pyppmd = "0.17.4"
-crowdstrike-falconpy = "^1.0.10"
+crowdstrike-falconpy-dev = "^1.1.4"
```

Comment:
I'd prefer to merge this once the features we need in `crowdstrike-falconpy` are in prod

Reply:
Certainly, just couldn't develop the OnlineState functionality without having the latest.  Before this merges to main we can swap it over to the production package of 1.1.4 (which releases Tuesday)

Reply:
Excellent! I can't see this getting merged on a Friday evening anyway, and with the US out on Monday for Juneteenth I think it's fair to say that this can be a Tuesday problem once `crowdstrike-falconpy` 1.1.4 is out in the wild :)